### PR TITLE
Feature/add details button custom renderer for grid component

### PR DESCRIPTION
**Pull request by kevmodrome**
## Description
Adds a details column to the grid component that links to the detail view. App builder needs to enter a url that is prepended to the id.

## Screenshots
https://cln.sh/fD4u5B

### CODE REVIEW

**Comment by joebudi** [comment]
@kevmodrome maybe make the icon var(--grey-6) to match the relationship tabs? If you feel blue is better, try var(--blue).

**Comment by kevmodrome** [comment]
Nah, whichever works for me! It was just blue because that's the link color, I changed it to the grey you suggested

File: packages/builder/src/components/userInterface/temporaryPanelStructure.js [hunk 1]
```diff
@@ -327,6 +327,12 @@ export default {
                 key: "datasource",
                 control: TableViewSelect,
               },
+              {
+                label: "Table URL",
+                key: "detailUrl",
+                placeholder: "tableName",
+                control: Input,
```
Comment (aptkingston) [inline packages/builder/src/components/userInterface/temporaryPanelStructure.js:334]: We actually have the `ScreenSelect` control made specifically for selecting screens to link to - that's what's used for choosing the target of a `Link` or a `Card` for example. Might be a good idea to use that here to make it easier for people to pick the right URL.

**Review by aptkingston — CHANGES_REQUESTED** [review]
Nice work! 👌 Just maybe change the control over like I noted above.

**Review by aptkingston — APPROVED** [review]

**Pull request merged**